apps/desktop-wallet/src/App.tsx

```diff
@@ -253,7 +252,6 @@ const App = () => {
       } else if (verifiedFungibleTokenSymbols.uninitialized.length > 0) {
         const symbols = verifiedFungibleTokenSymbols.uninitialized
 
-        dispatch(syncTokenCurrentPrices({ verifiedFungibleTokenSymbols: symbols, currency: settings.fiatCurrency }))
         dispatch(syncTokenPriceHistories({ verifiedFungibleTokenSymbols: symbols, currency: settings.fiatCurrency }))
```

Comment:
But we still need to get historic prices for the chart. Once that is removed, this whole useEffect will be deleted.